### ISSUE TITLE
Missing configurations for xpack

### ISSUE THREAD

**Issue by Ph3n1x** [excelsior-oss/excelsior-jet-api#11]
From the CLI help of xpack I could gather the info, that there is no possibility (or maybe just undocumented?) to configure all the nice MISC settings (like file associations, shortcuts, etc.).

```
 Excelsior Installer specific options:

 -company <string>               - set company name
 -product <string>               - set product name
 -version <version number>       - set product version
 -eula <pathname>                - specify the license agreement file (ANSI)
 -unicode-eula <pathname>        - specify the license agreement file
                                                                   (UTF-16LE)
 -splash <pathname>              - specify a splash image (in BMP format)
```

It would be nice to can configure all these settings through CLI parameters! And when that is possible, the gradle/maven plugin should also be able to configure them!

**Comment by pjBooms** [comment]
All advanced Excelsior Installer configuration settings will be supported in forthcoming Excelsior JET 11.3. You may try EA [now](https://www.excelsiorjet.com/beta). Once it's released (early November) we are going to support its new xpack interface in the plugins as well. So stay tuned.

**Comment by pjBooms** [comment]
Fixed in 0.9.5 release of plugins
